docs/release_notes.md

```diff
@@ -1,5 +1,39 @@
 # Release Notes
 
+## Version 1.1.0
+
+### Core
+
+- Attachment to geometry constraints to allow muscles to be attached to any given geometry.
+- Slide on geometry constraints to allow muscles to slide on any given geometry.
+- Mass support in muscle solvers to provide more control over the dynamics of the simulation.
+
+### Maya
+
+- Added AdnSkinMerge deformer to blend the results from animated and simualted input geometries.
+- Added support for attachment to geometry in AdnMuscle and AdnRibbonMuscle solvers.
+- Added support to slide on geometry in AdnMuscle and AdnRibbonMuscle solvers.
+- Added mass attributes in AdnMuscle and AndRibbonMuscle deformers to provide more control over the dynamics of the simulated object.
+- Added the ability to prevent the debugging of features if the geometry is not visible, hidden or isolated.
+- Added a sanity check to *Max Sliding Distance* attribute to warn the user if the input value is too high for the given target mesh.
+- Updated debugger system to visualize attachment to geometry constraints, slide on geometry constraints and sliding surface.
+- Added *Point Mass Mode* to AdnSimshape, AdnSkin, AdnMuscle and AdnRibbonMuscle which allows to specify either a mass per point or a density value from which point masses will be computed.
```

Comment:
Please, move this below "Added mass attributes..."

Reply:
Done.